study.md

```diff
@@ -44,21 +48,44 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+I'm still a bit unclear on the exact details. From what I gather, view-state has to do with managing multiple states/versions of a single web page. This is necessary because every time the page is requested from the server it returns a brand new copy, wiping any information or controls that were on the page before it refreshes. View-state is a state management feature in the ASP.NET page framework used to "preserve page and control values between round trips to the Web server".
+
+I used the page "ASP.NET View State Overview" for this answer.
+https://msdn.microsoft.com/en-us/library/bb386448.aspx
+
+
 ```
```

Comment:
It sounds like you got this! The only thing to note that while 'view-state' is a specific part of ASP.net development, but in a more general sense it's referring to the data from the model that is being displayed to the user in the current view. So if we made an AJAX request and loaded data to our app, that data being displayed is the view state. Front end routers manage the view state based on interaction from the user.